dashboard/test/controllers/plc/user_course_enrollments_controller_test.rb

```diff
@@ -114,7 +114,7 @@ class Plc::UserCourseEnrollmentsControllerTest < ActionController::TestCase
   end
 
   test 'Can navigate to specific course with underscored url' do
-    other_course = create(:plc_course, name: 'Other Course')
+    other_course = create(:plc_course, name: 'Other Course', id: Plc::Course.maximum(:id).next)
```

Comment:
No need to override `id` with the same value as the default in the factory (below). This should not be necessary.